packages/runtime/src/widgets/Charts/DoughnutChart.tsx

```diff
@@ -29,6 +30,7 @@ export const DoughnutChart: React.FC<ChartProps> = (props) => {
         datasets: config?.data?.datasets as ChartDataSets[],
       }}
       options={getMergedOptions(options, values?.title, config?.options)}
+      plugins={[ChartDataLabels as any]}
```

Comment:
Can we update this to also support inline plugins? Basically if an object is defined in the chart config we can also pass this through?